examples/node_cordon.rs

```diff
@@ -0,0 +1,42 @@
+#[macro_use] extern crate log;
+use futures::{StreamExt, TryStreamExt};
+use k8s_openapi::api::core::v1::Node;
+use kube::{
+    api::{Api, DeleteParams, ListParams, Patch, PatchParams, PostParams, ResourceExt},
+    runtime::{reflector, utils::try_flatten_applied, watcher},
+    Client,
+};
+use serde_json::json;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    std::env::set_var("RUST_LOG", "info,kube=debug");
+    env_logger::init();
+    let client = Client::try_default().await?;
+
+    let node_name = "node1";
+    let first_node = serde_json::from_value(json!({
```

Comment:
i'd call this something like `fake_node` to illustrate that this will never receive real use